### DISCUSSION TITLE
Request for Godot 4 Support in GitHub Copilot

### DISCUSSION THREAD

**Discussion by TheOneWayTruth** [community/community#67902] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Dear GitHub Copilot Team,

I'm writing to request support for Godot 4 in GitHub Copilot. The engine's popularity is on the rise as developers look for cost-effective alternatives to proprietary engines like Unity. However, Copilot currently focuses on Godot 3, while the syntax and features have significantly evolved in Godot 4.

Adding Godot 4 support would empower developers to work more efficiently and stay up-to-date with the latest Godot practices. It would also align Copilot with the growing trend of Godot adoption.

I understand this may require resources, but it's an investment that would greatly benefit the developer community and Godot's future success.

Thank you for considering this request.

Best Regards,
TheOneWayTruth

**Comment by Robert-K** [comment]
Currrently working on Godot 4 projects with Copilot enabled is such a pain. It constantly suggests incorrect syntax (`condition ? true : false`), deprecated methods (`.instance()`) and generally performs much much worse than I'm used to (writing comments, completing simple logic).

It's gotten to the point where I rather disable it for Godot 4 projects entirely 😢
